runtime/handle.js

```diff
@@ -134,23 +134,23 @@ class Collection extends Handle {
    * throws: Error if this view is not configured as a writeable view (i.e. 'out' or 'inout')
      in the particle's manifest.
    */
-  store(entity) {
+  async store(entity) {
     if (!this.canWrite)
       throw new Error("View not writeable");
     var serialization = this._serialize(entity);
-    return this._view.store(serialization);
+    return (await this._view.store(serialization));
```

Comment:
shouldn't need those outer brackets here and a couple below.

Reply:
Or an await :)